docs/fast_data/inputs_and_outputs.md

```diff
@@ -379,151 +459,120 @@ value: |
       }
     }
   }
-```
-
-</p>
-</details>
-
-**AsyncApi specification**:
-
-```yaml
-asyncapi: 2.4.0
-info:
-  title: Projection Update Producer
-  version: "1.0.0"
-channels:
-  projectionUpdateChannel:
-    subscribe:
-      message:
-        name: Projection Update
-        payload:
-          type: object
-          additionalProperties: false
-          properties:
-            key:
-              type: object
-            value:
-              type: object
-              additionalProperties: false
-              properties:
-                operationType:
-                  type: string
-                  enum: ["INSERT", "UPDATE", "DELETE", "UPSERT"]
-                operationTimestamp:
-                  type: number
-                documentId:
-                  type: string
-                projectionName:
-                  type: string
-                primaryKeys:
-                  type: Array
-                source:
-                  type: string
-                before:
-                  type: object
-                  additionalProperties: true
-                after:
-                  type: object
-                  additionalProperties: true
-                __internal__kafkaInfo:
-                  type: object
-                  additionalProperties: false
-                  properties:
-                    topic:
-                      type: string
-                    partition:
-                      type: integer
-                    offset:
-                      type: string
-                    key: {}
-                    timestamp:
-                      type: string
-          required: ["key", "value"]
+}
 ```
 
 #### Topic naming convention
-
-**producer** Real Time Updater
-
 `<tenant>.<environment>.<mongo-database>.<collection>.pr-update`
 
-An example:
+Example: `test-tenant.PROD.restaurants-db.reviews-collection.pr-update`
 
-```sh
-test-tenant.PROD.restaurants-db.reviews-collection.pr-update
-```
+## Single View
 
-### Projection Change
+This section covers the inputs and outputs concerning the Single View's aggregation.
 
-A `Projection Change` is an event that informs the listener that a Single View should be updated.
-It can be stored on either MongoDB or Kafka, depending on your architecture.
+### Projection Changes
 
-Its value has the following fields:
+**System**: MongoDB
```

Comment:
Can also be sent to kafka (Gio will provide you with some examples) with the GENERATE_KAFKA_PROJECTION_CHANGES env var